common-files/.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,17 @@
+# Automates creation of Release Drafts using Release Drafter
+# More Info: https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
+      - uses: release-drafter/release-drafter@v5
```

Comment:
How will we keep this up to date?

Reply:
this is pinned to the major version, maintainers can bump this to v6 if it ever changes and they need the feature

Reply:
Given https://github.blog/2020-06-25-dependabot-now-updates-your-actions-workflows/ do we need to amend Dependabot config to get this to work?